### ISSUE TITLE
Couldn't find the xcbeautify binary

### ISSUE THREAD

**Issue by ladislas** [tuist/tuist#5572]
### What happened?

I installed tuist using brew

```
brew install tuist --no-quarantine
```

When using `tuist build` or `tuist test`, I have the following error

```
Couldn't find the xcbeautify binary.
```

But it works in Xcode.

Looking at the release zip files, `xcbeautify` binary is indeed absent from the latest release and replaced with `Package.swift`.

### How do we reproduce it?

Install tuist with brew and generate/build/test your poject.

### Error log

```
Couldn't find the xcbeautify binary.
```

<img width="749" alt="image" src="https://github.com/tuist/tuist/assets/2206544/c7e6c574-c0f4-478e-90e4-a5037dfeb4e9">

### macOS version

14.1.1

### Tuist version

3.32

### Xcode version

15

**Comment by ladislas** [comment]
this seems to be fixed in 3.32.1

brew pr on its way https://github.com/Homebrew/homebrew-cask/issues/160220

**Comment by pepicrft** [comment]
Thanks for bringing this up. We realized it soon after doing the released and went ahead with a hotfix. Apologies for the inconvenience.
